app-tasks/rf/src/rf/uploads/modis/create_geotiff.py

```diff
@@ -37,20 +66,13 @@ def create_geotiffs(modis_path, output_directory):
                      '-a_nodata', '-28672',
                      '-separate']
 
-    warp_command = [
-        'gdalwarp', '-co', 'COMPRESS=LZW',
-        '-co', 'TILED=YES',
-        '-t_srs', 'epsg:3857',
-        combined_tif_filepath,
-        warped_tif_path
-    ]
-
     # Generate overviews
-    overview_command = ['gdaladdo', warped_tif_path, '2', '4', '8', '16', '32']
+    overview_command = ['gdaladdo', cropped_tif_path, '2', '4', '8', '16', '32']
 
     # Create final tif with overviews
-    translate_cog_command = ['gdal_translate', warped_tif_path, '-co', 'TILED=YES',
-                             '-co', 'COMPRESS=LZW', '-co', 'COPY_SRC_OVERVIEWS=YES', cog_tif_filepath]
+    translate_cog_command = ['gdal_translate', warped_tif_path, '-a_nodata', '0',
```

Comment:
I think nodata values were being carried through before. `0` isn't the correct `nodata` here though. Up [above](https://github.com/raster-foundry/raster-foundry/pull/3548/files#diff-10a7bc387128eee592e320757be4edf2R66) it's set to `-28672` which sets the correct mask, etc.